README.md

```diff
@@ -1,17 +1,261 @@
-# Receiving Api Posts
+# Receiving API POSTs
 
 ## Objectives
 
-  * Understand how params comes in from POST requests
-  * Use those params to create resources
+  1. Understand how `params` comes in from POST requests.
```

Comment:
This objective is kind of vague. Maybe we could change it to something like "Explain how we can submit form data using AJAX and `jQuery.post`"?

Hrm, that's not great either. What I'm going for is something that's a bit more concrete than the student just understanding -- they should be able to do something, and it has something to do with AJAX, forms, and serializing the form data.